pkg/syncer/metadatasyncer.go

```diff
@@ -1066,6 +1082,14 @@ func csiPVCUpdated(ctx context.Context, pvc *v1.PersistentVolumeClaim,
 	var volumeHandle string
 	var err error
 	if metadataSyncer.coCommonInterface.IsFSSEnabled(ctx, common.CSIMigration) && pv.Spec.VsphereVolume != nil {
+
+		// If multi VC, skip it
+		if metadataSyncer.coCommonInterface.IsFSSEnabled(ctx, common.MultiVCenterCSITopology) {
+			log.Debugf("PVCUpdated: %q is a vSphere volume claim in namespace %q. VCP volumes are not supported in a multi VC setup. Skipping update",
```

Comment:
need to correct typo: "csiPVCupdate: %q is a ..."

Reply:
We do not use "csiPVCUpdate" in metadatasyncer code.